modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring route admission policy
+
+By default, {microshift-short} prevents routes in multiple namespaces from using the same hostname. To accommodate different use cases, you can use the configuration option below to disable the `namespaceOwnership` check. This allows routes to claim different paths of the same hostname across namespaces.
+
+.Prerequisites
+
+* You have created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. Update the {microshift-short} `config.yaml` file parameter in the `ingress.routeAdmissionPolicy.namespaceOwnership` field to `InterNamespaceAllowed` as shown in the following example to disable the check:
++
+.Example `config.yaml` route admission policy settings
+[source,yaml]
+----
+#....
+ingress:
+  routeAdmissionPolicy:
+    namespaceOwnership: InterNamespaceAllowed <1>
+# ....
+----
+<1> Default value. Allows routes to claim different paths of the same host name across namespaces.
+To enable the checks, you can update the parameter in the `ingress.routeAdmissionPolicy.namespaceOwnership` field to `Strict`.This will prevent routes in different namespaces
```

Comment:
```suggestion
```
This can be deleted. The checking is a backend/implementation behavior and I don't think it's relevant. The user side is that we're restricting or allowing access.

Reply:
@pacevedom Am I right on this? WDYT? Is it important for end users to know the system behavior, or just how to configure the `namespaceOwnership` value?